packages/eslint-plugin-pf-codemods/src/rules/v6/tileDeprecated/tile-deprecated.md

```diff
@@ -0,0 +1,17 @@
+### tile-deprecated [(#)](https://github.com/patternfly/patternfly-react/pull/10821)
+
+Tile has been deprecated. Running the fix flag will update your imports to our deprecated package, but we suggest using Card instead.
```

Comment:
More of a nit since I don't think we've typically linked in this way, but wonder if it'd be worth linking to the Card as tile example once it goes in (via the v6 staging site)